css/style.css

```diff
@@ -16,4 +15,314 @@ body {
+}
+.image-content {
+  background-image: url("../img/first-background.jpg");
+  background-size: cover;
+  padding: 15rem;
+  color: white;
+  text-align: center;
+}
+
+.container {
+  margin: 0 auto;
+  max-width: 100%;
+}
+
+.image-content p {
+  font-size: 20px;
+}
+
+/* Nav styling starts here */
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  padding: 2px;
+}
+
+.nav-links {
+  display: flex;
+  padding: 0;
+  list-style-type: none;
+}
+
+.nav-links a {
+  color: gray;
+  padding: 10px 20px;
+  text-decoration: none;
+}
+
+.store {
+  cursor: pointer;
+}
+
+.nav-links li :hover {
+  color: darkorange;
+}
+
+/* section styling starts here */
+.how-container {
+  display: flex;
+  justify-content: space-around;
+  flex-direction: row;
+  align-items: center;
+}
+
+.how-it-works {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  /* padding: 6rem;*/
+  /* margin: 0.5rem; */
+}
+
+.title {
+  text-align: center;
+  margin-top: 6rem;
+  font-size: 25px;
+}
+
+.feature-container{
+  display: flex;
+  justify-content: space-around;
+  margin-top: 100px;
+  height: 439px;
+}
+
+.feature-img{
+  background-color: rgba(224, 99, 58, 0.1);
+  width:53.2%;
+  text-align: center;
+}
+
+.feature-img p{
+  margin-top: 100px;
+  font-size: x-large;
+  font-style: italic;
+  line-height: 35px;
+}
+
+.feature-img button{
+  background-color:darkorange ;
+  padding:15px 30px;
+  border-style: none;
+  border-radius: 5px;
+  
+}
+
+.feature-img button a {
+  text-decoration: none;
+  color: white;
+
+}
+
+/* button styling starts here */
+.btn {
+  background-color: darkorange;
+  color: white;
+  padding: 10px 30px;
+  /*margin-left: 200px;*/
+  margin-top: 50px;
+  border-radius: 5px;
+  cursor: pointer;
+  border-style: none;
+}
+
+/* footer styling starts here */
+
+footer {
+  background: rgb(255, 249, 249);
+  padding: 10px;
+}
+
+.help-links {
+  list-style-type: none;
+  display: flex;
+  justify-content: center;
+  padding: 0;
+}
+.help {
+  text-align: center;
+  padding: 5px;
+  margin-top: 50px;
+}
+
+.socials {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  /* margin-top: 50px; */
+}
+.socials li {
+  width: 40px;
+  height: 40px;
+  border: 1px solid lightgray;
+  border-radius: 50%;
+  margin-left: 10px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+.copyright {
+  text-align: center;
+  opacity: 0.2;
+  font-size: 12px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-left: 20px;
+}
 
+/* main styling */
+
+ .wrapper {
+  display: flex;
+  flex-direction: row;
+  
+
+}
+
+/* styling for image on form page */
+.img-container {
+  display: flex;
+  flex-direction: column;
+   width: 50%;
+   margin-left:50px ;
+  
+}
+
+
+/* styling for form */
+
+h2 {
+  color:#E0633A;
+  font-size: xxx-large;
+
+}
+
+form{
+  width: 50%;
+  height: auto; 
+  display: flex;
+  flex-direction: column;
+  margin: 30px 15px;
```

Comment:
Don't use margin to center! Use flex!
The form and the image should be in a flex row together so you can center them together. Then focus on turing the form into its own container, which will be a column! (with maybe some more rows inside it).